test/testdrive/bytes.td

```diff
@@ -60,7 +60,7 @@ named_col  false     bytea
 > SELECT * FROM data_offset
 data           offset
 ------------------------
-"\\xc2\\xa92"  1
+"\\xc2blablasomeerror\\xa92"  1
```

Comment:
Slipped in? 😀

Reply:
Nope, that was for testing. Is this how it's supposed to look?
![Screenshot 2024-02-01 at 15 09 52](https://github.com/MaterializeInc/materialize/assets/2335377/2657b90b-dcc2-4916-8d22-11aeb59d9369)

Reply:
Here are some examples how it looks like (not all of the are testdrive instances): https://buildkite.com/materialize/tests/builds/73749

However, I unfortunately don't have a comparison error but only syntax errors included.

Let me see if I can still find a build...

Reply:
I haven't found one but I think it should be ok.